sdk/monitor/monitor-opentelemetry-exporter/src/sampling/samplingUtils.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Some indication that this is an implementation of the DJB2 hash algorithm in this file could be helpful.